Light.json

```diff
@@ -1358,6 +1359,7 @@
         "darkSideIcons": 1,
         "gametext": "Light:  If Luke here or out of play, Force Projection is [Immune to Sense].  Dark:  While Luke here, attrition against you everywhere is +1."
       },
+      "rules": [{"name":"Generic Systems Deployment", "url":"https://www.starwarsccg.org/location-deployment-specifics-general"}],
```

Comment:
Looks like the "Generic" rules were removed for the Dark cards, but are still present for the light-side.  I'm guessing you are still working on this (this is mostly a note for whoever reviews this later to make sure to check both files)